src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/Client/MainLayout.razor

```diff
@@ -0,0 +1,7 @@
+﻿@inherits LayoutComponentBase
+
+<div class="page">
+    <main>
+        @Body
+    </main>
+</div>
```

Comment:
Do we need the `div` here? Does it work to reduce it to the following?

```razor
@inherits LayoutComponentBase

<main>@Body</main>
```

Reply:
Nope, looks like we do not need the `div`. I can remove them.